src/homework-1/task2/task-2.js

```diff
@@ -0,0 +1,19 @@
+const fs = require('fs')
+const csv = require('csvtojson')
+
+const readStream = fs.createReadStream('./resources/books.csv')
+const writeStream = fs.createWriteStream('./resources/books.txt')
+
+csv()
+  .fromStream(readStream)
+  .on('data', data => writeStream.write(data, 'utf-8'))
+  .on('done', error => processFinishedHandler(error))
```

Comment:
The problem is this callback will handle only errors from transform stream, on the stage when `readStream` paths some chunks into `csv` stream and `csv` stream tries to converts those chunks to `json`. But you have no handlers for errors which may occur in the readStream during reading `.csv` file or in the writeStream during writing `.txt` file.  You have to use: 
```
readStream.on("error", (err => {}))
writeStream.on("error", (err) => {});
```
It's optional, but you can also pay attention on `require("stream").pipeline` method, it can help to resolve this task in a more pretty way :)

Reply:
Thank you.  Fixed with pipeline